packages/marshal/test/test-rankOrder.js

```diff
@@ -5,6 +5,7 @@ import { test } from './prepare-test-env-ava.js';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { fc } from '@fast-check/ava';
 import { makeTagged } from '@endo/pass-style';
+import { arbPassableKit } from '@endo/pass-style/tools.js';
```

Comment:
You could revert to exporting the items by name in this new form, to avoid the separate destructure.

Reply:
Good point. Done